servers/cromwell/jobs/test/test_workflows/stress_test/inputs.json

```diff
@@ -0,0 +1,21 @@
+{
+  "StressTestAdapterSmartSeq2SingleCell.gtf_file": "gs://hca-dcp-mint-test-data/reference/GRCh38_Gencode/gencode.v27.primary_assembly.annotation.gtf",
+  "StressTestAdapterSmartSeq2SingleCell.hisat2_ref_trans_name": "gencode_v27_trans_rsem",
+  "StressTestAdapterSmartSeq2SingleCell.rrna_intervals": "gs://hca-dcp-mint-test-data/reference/GRCh38_Gencode/gencode.v27.rRNA.interval_list",
+  "StressTestAdapterSmartSeq2SingleCell.hisat2_ref_index": "gs://hca-dcp-mint-test-data/reference/HISAT2/genome_snp_tran.tar.gz",
+  "StressTestAdapterSmartSeq2SingleCell.genome_ref_fasta": "gs://hca-dcp-mint-test-data/reference/GRCh38_Gencode/GRCh38.primary_assembly.genome.fa",
+  "StressTestAdapterSmartSeq2SingleCell.hisat2_ref_trans_index": "gs://hca-dcp-mint-test-data/reference/HISAT2/gencode_v27_trans_rsem.tar.gz",
+  "StressTestAdapterSmartSeq2SingleCell.rsem_ref_index": "gs://hca-dcp-mint-test-data/reference/GRCh38_Gencode/gencode_v27_primary.tar",
+  "StressTestAdapterSmartSeq2SingleCell.gene_ref_flat": "gs://hca-dcp-mint-test-data/reference/GRCh38_Gencode/GRCh38_gencode.v27.refFlat.txt",
+  "StressTestAdapterSmartSeq2SingleCell.hisat2_ref_name": "genome_snp_tran",
+  "StressTestAdapterSmartSeq2SingleCell.stranded":"NONE",
+  "StressTestAdapterSmartSeq2SingleCell.reference_bundle": "bf51d668-3e14-4843-9bc7-5d676fdf0e01",
+  "StressTestAdapterSmartSeq2SingleCell.format_map": "gs://hca-dcp-mint-test-data/adapters/file_format_map.json",
+  "StressTestAdapterSmartSeq2SingleCell.dss_url": "https://dss.dev.data.humancellatlas.org/v1",
+  "StressTestAdapterSmartSeq2SingleCell.submit_url": "http://api.ingest.dev.data.humancellatlas.org/",
```

Comment:
How about replacing all the actual urls in this file with fake url strings? Although there are no credentials, we may want to limit the user groups to our public bucket? (Even just for reducing egress cost lol)

Reply:
That's a good point! I think it would be useful to keep at least one url in case we wanted to test the behavior of the link in the inputs list on the details page. What do you think?

Reply:
You are right, it is better to have one realistic url case.